dracut/anaconda-hmcroot

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# anaconda-hmcroot:
+# Provide access to the root on the USB/DVD drive
+# of the SE/HMC of System z hardware.
```

Comment:
I think the comment could be just one line.

Reply:
Fixed.